pkg/operator/operator.go

```diff
@@ -130,10 +130,31 @@ func (o *Operator) Run(stopc <-chan struct{}) error {
 	}
 
 	go o.worker()
-
 	go o.cmapInf.Run(stopc)
 
-	<-stopc
+	ticker := time.NewTicker(5 * time.Minute)
+	defer ticker.Stop()
+
+	time.Sleep(10 * time.Second)
```

Comment:
Normally we can use waitforcachesync here, however let's refactor it later if tests are passing right now